ProjectSidewalk/SidewalkWebpage
# Resolves #965: clears temporary label ID after closing context menu

Resolves #965 

This fixes logs still having a temporary label ID after the context menu is closed using keyboard shortcuts:
- Force description-box blur on context menu hide
- Re-order context menu events to make ContextMenu_Close... come last

The issue is related to (but doesn't fix) #1256 because it was caused by events being in the wrong order. KeyboardShortcut... events were being fired after ContextMenu_Close... events, which set the temporary label ID back.

Here are the logs from using the fixed context menu 5 times, each closing the menu a different way (clicking outside, pressing ENTER, pressing ESC, clicking the X, clicking "Ok"):
![1](https://user-images.githubusercontent.com/7133966/52528008-a57ab400-2c87-11e9-848e-bb929972ab83.png)
![2](https://user-images.githubusercontent.com/7133966/52528011-a7dd0e00-2c87-11e9-9c10-f8488f0e1cdc.png)
![3](https://user-images.githubusercontent.com/7133966/52528012-a9a6d180-2c87-11e9-845e-0843211d4434.png)
![4](https://user-images.githubusercontent.com/7133966/52528013-aad7fe80-2c87-11e9-883a-ca0d7103a371.png)
![5](https://user-images.githubusercontent.com/7133966/52528015-aca1c200-2c87-11e9-9027-4f68abcabf13.png)


To test:
- Ensure that logs are working as normal
- Ensure that logs that didn't happen inside the context menu don't have a temporary label ID
